src/contracts/SafetyNet.sol

```diff
@@ -350,7 +360,8 @@ contract SafetyNet is ISafetyNet, ReentrancyGuard, OwnableUpgradeable {
 
     for (uint256 epochIndex = 0; epochIndex < currentEpochIndex; epochIndex++) {
       for (uint256 i = 0; i < safetyNet.members.length; i++) {
-        if (!epochMemberDeposits[_safetyNetId][epochIndex][safetyNet.members[i]]) {
+        address m = safetyNet.members[i];
```

Comment:
same comment here about abbreviated variable names. would suggest changing this to `_member`